docs/pages/admin-guides/deploy-a-cluster/deploy-a-cluster.mdx

```diff
@@ -6,10 +6,12 @@ description: "Guides to running a self-hosted Teleport cluster in production."
 These guides show you how to run a self-hosted Teleport Enterprise cluster in
 production.
 
-## Dedicated account dashboard
+# Dedicated account dashboard
```

Comment:
```suggestion
## Dedicated account dashboard
```
This should be an H2 for SEO purposes (there's already an H1 added based on the title)